src/inference/src/ie_network_reader.cpp

```diff
@@ -549,16 +552,19 @@ CNNNetwork details::ReadNetwork(const std::string& model,
         params.emplace_back(weights_buffer);
     }
 
-    FE = manager.load_by_model(params);
-    if (FE) {
-        FE->add_extension(ov_exts);
-        if (!exts.empty())
-            FE->add_extension(wrap_old_extensions(exts));
-        inputModel = FE->load(params);
-    }
-    if (inputModel) {
-        auto ngFunc = FE->convert(inputModel);
-        return convert_to_cnnnetwork(ngFunc, exts, newAPI, frontendMode);
+    try {
+        FE = manager.load_by_model(params);
+        if (FE) {
+            FE->add_extension(ov_exts);
+            if (!exts.empty())
+                FE->add_extension(wrap_old_extensions(exts));
+            inputModel = FE->load(params);
+        }
+        if (inputModel) {
+            auto ngFunc = FE->convert(inputModel);
+            return convert_to_cnnnetwork(ngFunc, exts, newAPI, frontendMode);
+        }
+    } catch (...) {
```

Comment:
why PDPD is selected? I suppose we need to understand this thing instead of catching exception blindly.